frontend/src/pages/HomePage.tsx

```diff
@@ -19,36 +17,38 @@ const headersData = [faq, review];
 
 const HomePage = (): ReactElement => {
   return (
-    <Container className={styles.Home}>
-      <AppBar headersData={headersData} />
-      <div className={styles.homepageDescription}>
-        <h5>Search for off-campus housing, review apartments, and share feedback!</h5>
-      </div>
-      <SearchBar placeholder="Search by any location e.g. “301 College Ave" ariaLabel="" />
-      <SectionDivider />
-      <Row>
-        <Col xs={12} lg={4}>
+    <Box style={{ background: '#f6f6f6' }}>
```

Comment:
Avoid inline styles, we can use the `withStyles` hook by MUI for MaterialUI component styles and external stylesheet files for other components.